.travis.yml

```diff
@@ -59,7 +59,7 @@ env:
     - PGVERSION=9.0 PQTEST_BINARY_PARAMETERS=no
 
 script:
- - go test -v ./...
+ - result=$(gofmt -d -e $(find -name \*.go)); test -z "$result" || (echo "$result" && false) && go test -v ./...
```

Comment:
Indeed! The pipe character is significant in YAML. You can use the [folded style](http://www.yaml.org/spec/1.2/spec.html#id2796251) for this:

```yaml
script: >
  goimports -d -e $(find -name \*.go) | awk '{ print } END { exit NR == 0 ? 0 : 1 }' && go test -v ./...
```